src/legacy/core_plugins/kibana/public/management/sections/index_patterns/create_index_pattern_wizard/components/step_time_field/step_time_field.js

```diff
@@ -109,10 +118,45 @@ export class StepTimeFieldComponent extends Component {
     }));
   }
 
-  createIndexPattern = () => {
+  createIndexPattern = async () => {
+    const { createIndexPattern } = this.props;
     const { selectedTimeField, indexPatternId } = this.state;
     this.setState({ isCreating: true });
-    this.props.createIndexPattern(selectedTimeField, indexPatternId);
+    try {
+      await createIndexPattern(selectedTimeField, indexPatternId);
+    } catch (error) {
+      if (!this.mounted) return;
```

Comment:
Can you explain the case when an error is thrown but the component isn't mounted?

Reply:
1. User opens the form, component is mounted.
2. User clicks "Save" button, `createIndexPatter` async request is dispatched.
3. While request is in progress, user navigates to some other form in management app, so this component is unmounted.
4. Async request returns and throws `"Conflict"`.
5. But component is already unmounted because user navigated to a different form.